app/models/ability.rb

```diff
@@ -66,7 +66,7 @@ def signed_in(user)
 
     can [:new, :create], Registration do |registration|
       conference = registration.conference
-      conference.registration_open? && !conference.registration_limit_exceeded? || conference.program.speakers.confirmed.include?(user)
+      conference.registration_open? && (!conference.registration_limit_exceeded? || conference.program.speakers.confirmed.include?(user))
```

Comment:
As I just wrote in https://github.com/openSUSE/osem/issues/1827#issuecomment-348426404, this is made in purpose. The expected condition is the one we already have:

``` Ruby
(conference.registration_open? && !conference.registration_limit_exceeded?) || conference.program.speakers.confirmed.include?(user)
```

Please remove the first commit 😄